sqlmesh/core/context.py

```diff
@@ -1571,6 +1572,13 @@ def _has_environment_changed() -> bool:
 
         return success
 
+    def clear_caches(self) -> None:
+        for path in self.configs.keys():
+            self._clear_cache(path)
+
+    def _clear_cache(self, path: Path) -> None:
+        rmtree(path / c.CACHE)
```

Comment:
Do we even need this method? Should we just call `rmtree` directly above?

Reply:
Removing it is cleaner :)